.circleci/config.yml

```diff
@@ -68,6 +68,25 @@ jobs:
     <<: *job_defaults
     steps:
       - checkout
+      - run:
+          name: Merge to target branch
```

Comment:
You're going to have to do this to very single stanza where it has `checkout`.

Reply:
Done, using commands.